include/EVT/dev/LCD.hpp

```diff
@@ -120,7 +118,17 @@ class LCD {
      * @param column[in] the column to write the text too. Range 0-127.
      * @param wrapText[in] whether or not the text should be wrapped around the edge of the screen.
      */
-    void writeText(const char* text, uint8_t page, uint8_t column, bool wrapText);
+    void writeSmallText(const char* text, uint8_t page, uint8_t column, bool wrapText);
```

Comment:
I can see that the implementation is relatively different for writing small and large text, but I feel like, from an interface standpoint, it would make more sense to have a single `writeText()` method and then set the font size somewhere else. I think I would prefer to have weird behavior if used improperly than make the application code explicitly say the font size every time it writes text on the screen. Does that make sense?

Reply:
That makes sense, I just condensed the function into a single `writeText()` that allows us to select the font size during the call